travis_script.sh

```diff
@@ -22,13 +22,17 @@ case $TEST_SUITE in
   features)
     bundle exec rspec --format documentation spec/features
     ;;
+  requests)
```

Comment:
That's a bit much setup just for one test. Please also run this in the 'controllers' section below. Like

`bundle exec rspec --format documentation spec/controllers spec/requests`

Reply:
little late, but done :)
thanks!